gr-analog/python/analog/qa_fastnoise.py

```diff
@@ -30,21 +31,58 @@ def setUp (self):
     def tearDown (self):
         self.tb = None
 
-    def test_001(self):
-        # Just confirm that we can instantiate a noise source
-        op = analog.fastnoise_source_f(analog.GR_GAUSSIAN, 10, 10)
+    def test_001_test_moments(self):
```

Comment:
In a perfect world one would have a unit test for each type of source or at least one for real and one for complex sources. But this is fine for now.